configure.ac

```diff
@@ -304,7 +304,7 @@ AC_SUBST([ENABLE_COVERAGE], [$enable_coverage])
 AC_ARG_ENABLE([finject],
         AS_HELP_STRING([--disable-finject],[disable fault injection support]),
         [],
-        [enable_finject=$enable_dev_mode]
+        [enable_finject=$enable_release]
 )
 AS_IF([test x$enable_finject = xyes],
       AC_DEFINE([ENABLE_FAULT_INJECTION]))
```

Comment:
Can we add a ST to test FIS and show how to use the m0console to enable/disable a FI point?

Reply:
We already make use of FI in ST's,
$ git grep -rn "m0console" m0t1fs/
m0t1fs/linux_kernel/st/m0t1fs_rconfc_fail_test.sh:120:  env "${TRACE_console:-}" "$M0_SRC_DIR"/console/m0console \

Reply:
OK.